src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -152,13 +152,11 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
     }
 
     private PullRequest getPullRequest(String payload, GitHub gh) throws IOException {
-        PullRequest pr = gh.parseEventPayload(new StringReader(payload), PullRequest.class);
-        return pr;
+        return gh.parseEventPayload(new StringReader(payload), PullRequest.class);
```

Comment:
I was wondering what the reason behind this was.  I'll change it back and suppress the warnings for these.